styles.css

```diff
@@ -1 +1,751 @@
+}
+
+.partners-hr {
+  color: #ec5242;
+  width: 4%;
+  border: 1px solid #ec5242;
+  margin: 10px;
+  margin-bottom: 20px;
+}
+
+h2.title2 {
+  font-size: 1.5rem;
+  width: 20%;
+  min-width: 60px;
+  color: #ec5242;
+  font-family: 'COCOGOOSE', sans-serif;
+  overflow: visible;
+}
+
+.join-btn {
+  display: block;
+  background-color: #ec5242;
+  color: #fff;
+  padding: 20px 30px;
+  font-weight: 600;
+  border: none;
+  outline: none;
+  cursor: pointer;
+  margin: 20px auto;
+  margin-bottom: 35px;
+}
+
+.speakers {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  margin: 0 5%;
+}
+
+h3,
+h3.program {
+  margin-top: 30px;
+}
+
+img.event1,
+img.event2 {
+  width: 100%;
+  height: 0%;
+  margin: 20px 0;
+}
+
+section.partners {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  padding: 20px 20px;
+  background-color: #424146;
+}
+
+img.partner1,
+img.partner2,
+img.partner3,
+img.partner4,
+img.partner5 {
+  width: 15%;
+}
+
+div.partners-logos {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: space-between;
+  width: 100%;
+  padding: 20px 0;
+}
+
+footer {
+  display: flex;
+  flex-direction: row;
+  background-color: white;
+  align-items: center;
+  justify-content: space-around;
+  height: 200px;
+  overflow: hidden;
+}
+
+div.footer-logo {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: center;
+  padding: 20px 0;
+}
+
+p.copyright {
+  font-size: 24px;
+  color: black;
+  padding: 20px 0;
+}
+
+p.copyright-about {
+  font-size: 24px;
+  color: white;
+  padding: 20px 0;
+}
+
+div.social-links {
+  background-color: #231f20;
+  display: flex;
+  flex-direction: row;
+  justify-content: end;
+  padding: 5px;
+}
+
+a.social {
+  color: white;
+  padding-right: 60px;
+}
+
+.partners-container {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-around;
+  align-items: center;
+  gap: 5px;
+}
+
+section.about-section {
+  background-image: url("./Images/background-opacity.png");
+  background-size: cover;
+  background-repeat: no-repeat;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  padding: 20px 20px;
+  background-color: white;
+}
+
+section.past-graduates {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  background-color: white;
+}
+
+p.about-us-para {
+  margin: auto;
+  margin-top: 5%;
+  line-height: 1.5;
+  background-color: white;
+  padding: 5%;
+}
+
+h1.about-us-title {
+  font-size: 2.5rem;
+  color: #ec5242;
+  text-align: center;
+}
+
+img.my-aboutus-logo {
+  width: 30%;
+  margin: 25px auto;
+  border: 1px solid #d3d3d3;
+  padding: 10px;
+}
+
+div.graduates-images {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  width: 70%;
+  margin: 30px auto;
+  margin-bottom: 15px;
+  object-fit: contain;
+}
```

Comment:
Great design @waseemqureshi2020 🤝

- [ ] Kindly add a blur to your images so that they appear like in the design that was provided. Implement this on both desktop and mobile versions of the about section. It would also be better to use two images instead of three so as to match the design given.
- Your design
![about-blur](https://user-images.githubusercontent.com/63917725/197341626-b4a084a4-cc95-4e60-9796-612fbf45d706.PNG)
- Design Provided
![blur-design](https://user-images.githubusercontent.com/63917725/197341687-f84cdbc0-f5bb-4b5f-9517-2cb116d6f3ac.PNG)

Reply:
I have used the blur effect as you requested. But I think it's not always compulsory to copy 100% of the design given, As a designer, we can use some creative ideas to make the website look more beautiful.